wdl/TrainSRGenotyping.wdl

```diff
@@ -97,7 +97,7 @@ task GenotypeSRPart1 {
 
   RuntimeAttr default_attr = object {
     cpu_cores: 1, 
-    mem_gb: 3.75,
+    mem_gb: 6,
```

Comment:
Assume this is necessary now, but any idea what's using more memory?

Reply:
It's been a little while since I made this change, but I think I remember most of the details. I didn't investigate very fully, in part because it takes awhile to get to this part of the pipeline, so I tend to favor solutions that kill problems dead, rather than screwing around with long-shots.

1. This calls SR_genotype.opt_part1.sh, which has a few algorithms that could be implemented in a much more memory efficient manner. The location of the crash I was getting was the section on lines 144-152, which constructs a long list of patterns to search, and then does fgrep -wf to search them. I considered trying to rewrite this section to be more memory efficient, but it would alter the sv_pipeline docker (invalidating my call cache) and make reviewing harder. So I bumped up memory instead because it was a small task and didn't take all that long to complete (maybe around 5-10 minutes if I recall correctly).
2. As for why it's using more memory now than it used to, I'd guess we were previously pretty close to the line (there's no VM scaling in this task) and my fixes to the BAF processing earlier in the pipeline are retaining more variants.

Reply:
As long as it doesn't increase cost substantially that's fine